src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +48,21 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        Set<String> lowerCaseSet = changeStringsToLowerCase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            final Set<String> wordsInName = changeStringsToLowerCase(new HashSet<>(person.getName().getWordsInName()));
```

Comment:
A possible better name for this method is "getLowerCaseStrings", ponder on why that might be the case.